### ISSUE TITLE
Permission denied - runnig is as a normal user [SOLVED]

### ISSUE THREAD

**Issue by chrisfxwolf** [netblue30/firejail#3841]
Hi, 
I moved from KDE Neon to openSUSE. I'm not an advanced user. 
Firejail wont start as a $ (normal user).
Example

$firejail --version
$bash: /usr/bin/firejail: Permission denied

however:

$ ls -al /usr/bin/firejail
-rwsr-x--- 1 root firejail 388840 Aug 11 16:46 /usr/bin/firejail

I installed firejail form opensuse repositories, list of installing used zypper:

1 new package to install.
Overall download size: 516.5 KiB. Already cached: 0 B. After the operation, additional 1.5 MiB will be used.
Continue? [y/n/v/...? shows all options] (y): Retrieving package firejail-0.9.62-lp152.3.3.1.x86_64 (1/1), 516.5 KiB (  1.5 MiB unpacked)
Retrieving: firejail-0.9.62-lp152.3.3.1.x86_64.rpm [done]

Checking for file conflicts: [.......done]
(1/1) Installing: firejail-0.9.62-lp152.3.3.1.x86_64 [............done]

Hence all looks all right. I was reading documentation and done steps. But...

My system

Firejail instaled from openSUSE repositories

Operating System: openSUSE Leap 15.2
KDE Plasma Version: 5.18.6
KDE Frameworks Version: 5.71.0
Qt Version: 5.12.7
Kernel Version: 5.3.18-lp152.57-default
OS Type: 64-bit
Processors: 8 × Intel® Xeon® CPU W3530 @ 2.80GHz
Memory: 31.3 GiB of RAM

**Comment by smitsohu** [comment]
> $ ls -al /usr/bin/firejail
> -rwsr-x--- 1 root firejail 388840 Aug 11 16:46 /usr/bin/firejail

In openSUSE, Firejail can be used only by members of the firejail group. So if you are not a member of this group already, you will need to add yourself.

To check which groups you are a member of: `groups`
To add yourself to the firejail group: `sudo usermod -a -G firejail $USER`

**Comment by smitsohu** [comment]
Ah, forgot. You need to logout and login for the changes to take effect.

**Comment by SkewedZeppelin** [comment]
> You need to logout and login for the changes to take effect.

I tested Leap 15.2 the other day and needed a reboot to get it to apply for some
reason.

**Comment by chrisfxwolf** [comment]
As I understand I need:

# addgroup firejail
# chown root:firejail /usr/bin/firejail
# chmod 4750 /usr/bin/firejail

and add myself

# usermod -a -G firejail [ my username]

..and 

logout and login back or reboot

**Comment by SkewedZeppelin** [comment]
no, openSUSE package already adds the group and sets the correct permissions.


You should only need the command to add yourself to the group:
usermod -a -G firejail $user;

**Comment by chrisfxwolf** [comment]
Yes is working. I think so that original [documentation](https://firejail.wordpress.com/support/) have a mess.
Which say:

```
# addgroup firejail
# chown root:firejail /usr/bin/firejail
# chmod 4750 /usr/bin/firejail
```
This give me a mess in the groups. Which are done (as you mentioned) by firejail. 
As a result is:
```
$ ls -al /usr/bin/firejail
-rwsr-x--- 1 root firejail 388840 Aug 11 16:46 /usr/bin/firejail
```
# SOLUTION
* So after install firejail we need:
```
usermod -a -G firejail [username]
```

* ...and 
restart computer. Somebody mentioned to logout. In my case is not working ;-)

# 

Hence, this subject I consider it close.

Thank You guys!!!

Ps. 

However, it not an easy. Now Thunderbird is not working.
As well clementine icon in KDE panel is disappearing 

I'll be looking for  solution by myself.

**Comment by smitsohu** [comment]
Somewhat offtopic: There are a number of openSUSE related fixes in git which are not contained in 0.9.62. So if following git is an option I would actually recommend that.

Don't know if it would be possible to convince the openSUSE package maintainer to ship upcoming Firejail release 0.9.66 in Leap 15.3. It seems that openSUSE is somewhat less dogmatic about package upgrades than some other distributions.

**Comment by chrisfxwolf** [comment]
Hmm.. I think so Tumbleweed will have newest firejail. But T is not for me. I was using KDE Neon. Very good distro.
But, I need STABLE distro. That I was changing to openSUSE Leap. I'm not a gamer. And I don't care that some packages are old. However, I'm just considering to give up firejail which was easy to management on KDE Neon by firetool (which is not in suse repositories or flatpak). And learn apparmor. This little mess which firejail made (I made too) on my system (thanks suse for snapshot) is not what I expected. Ok, we finish this off-topic, before admin will kill us ;-).
